.github/pull_request_template.md

```diff
@@ -1,42 +1,3 @@
-<!--
-Not all PRs will require all tests to be carried out. Delete where appropriate.
--->
-
-<!--
-MODIFY THIS AFTER your new app repo is in https://github.com/giantswarm/github
-@team-halo-engineers will be automatically requested for review once
-this PR has been submitted. (But not for drafts)
--->
-
-This PR:
-
-- adds/changes/removes etc
-
-### Testing
-
-Description on how {APP-NAME} can be tested.
-
-- [ ] fresh install works
-  - [ ] AWS
-  - [ ] Azure
-  - [ ] KVM
-- [ ] upgrade from previous version works
-  - [ ] AWS
-  - [ ] Azure
-  - [ ] KVM
```

Comment:
I have checked some of the templated repos and none is using this part on the latest PR merged

Reply:
KVM is also not supported anymore... it seems.